source/process/spec-process.rst

```diff
@@ -0,0 +1,46 @@
+Spec Review Process
+===================
+
+This is a rough guideline for the process for drafting and reviewing specs. It’s meant to be used as a starting point, but not all specs will follow this exact process - please use your judgment, as it’s something that will vary depending on the situation.
+
+1. Brainstorm / research.
+    - Research topic
+    - Research customer requirements
+    - Pair design brainstorm
+    - User workflows
+    - Consider options / vision for UI
+2. Initial draft.
+    - Goal / “Why"
+
+      - Link to customer data / feedback
+      - Link back to company objectives
+    - User Stories
+    - Rough and Quick Wireframes / Mock ups
+
+      - Flows of screens
+      - Focus should be on the mock ups, with text that supports design
+
+    - Questions
+    - “Not in scope”
+    - Success Metrics
+    - Pair reviews
+
+      - Iterate on designs as needed
+      - Early check in with developers to identify any potential issues
+3. Share with team.
+    - Post draft with @channel in Spec Review channel, asking to review and add comments.
+4. Spec review.
+    - Schedule a meeting with interested people to discuss comments and any other questions about the spec:
+
+      - UX Design
+      - Product Manager
+      - Developer
+      - Test
+5. Update draft.
+    - Update based on feedback. Update wireframes / mock ups.
+6. Review again.
+    - Post updated draft in Spec Review channel, asking the people who previously commented for another review.
+    - Share with interested customers for feedback
+7. Repeat reviews/updates.
+    - Repeat as needed until final draft is signed off on.**
```

Comment:
Yeah I think the idea is that it should be signed off on by all groups, but I can update to make that more explicit